tools/test-case-generators/format-request-map.json

```diff
@@ -16,14 +16,9 @@
       "arch": "",
       "image-type": "azure-rhui",
       "repositories": [],
-      "filename": "disk.vhd",
+      "filename": "disk.vhd.xz",
```

Comment:
How did this work? What changed?

Reply:
My guess is we never generated the image info since the change?  I don't know.  My image info generation failed because it couldn't get the right file, so I have no idea how it worked before if it did.

Reply:
Actually, could you explain all the changes, my knowledge of how this works has been lost.

Reply:
No need, I think it was explained well enough why this is all ok :)